GitUI/UserControls/FileStatusList.cs

```diff
@@ -400,6 +400,19 @@ public IEnumerable<GitItemStatusWithParent> SelectedItemsWithParent
                 .Where(i => i.Group?.Tag is GitRevision)
                 .Select(i => new GitItemStatusWithParent(i.Group.Tag<GitRevision>(), i.Tag<GitItemStatus>()));
 
+        public IEnumerable<GitItemStatus> FirstGroupItems
+        {
+            get
+            {
+                if (FileStatusListView.Groups.Count == 0)
+                {
+                    return new List<GitItemStatus>();
+                }
+
+                return from ListViewItem item in FileStatusListView.Groups[0].Items select item.Tag<GitItemStatus>();
```

Comment:
Can we write it in a functional way?
```cs
return FileStatusListView.Groups[0].Items.Select(item => item.Tag<GitItemStatus>())
```

Do we need to check for nulls?

Reply:
ListViewItemCollection does not support Select()
This is what Re# suggested
I can write as a loop instead

Reply:
Do it as a `foreach` loop, it is faster and less allocations. I think it would look like:
```cs
foreach (ListViewItem item in FileStatusListView.Groups[0].Items)
{
	yield return item.Tag<GitItemStatus>();
}
```

Reply:
ok, rebased and pushed